scripts/circle_helper.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+function mapbox_time_start {
+    :
+}
+
+function mapbox_time_finish {
+    :
+}
+
+function mapbox_time {
+    :
+}
+
+function mapbox_fold {
+    :
+}
+
+function mapbox_nanoseconds {
+    :
+}
+
+export JOBS=${CIRCLE_NODE_TOTAL:-6}
+export -f mapbox_fold
+export -f mapbox_nanoseconds
+export -f mapbox_time
+export -f mapbox_time_start
+export -f mapbox_time_finish
```

Comment:
No real need in the final change - I've added this in the beginning to speed up the initial migration efforts - we can totally get rid of this 👍